SimpleNtuplizer/plugins/SNtup_PFClusters.cc

```diff
@@ -53,6 +53,11 @@ void SimpleNtuplizer::setPFVariables(const edm::Event& iEvent,
   iSetup.get< EcalMappingRcd >().get(ecalmapping);
   elecMap_ = ecalmapping.product();
 
+  ///rho
+  edm::Handle< double > rhoH;
+  iEvent.getByToken(rhoToken_,rhoH);
+  rho_pf = *rhoH;
```

Comment:
I don't think this will work:

fixedGridRhoFastjetAll depends on pfCandidates:

http://cmslxr.fnal.gov/source/RecoJets/JetProducers/python/fixedGridRhoProducerFastjet_cfi.py#0005

but pfCandidates cannot be formed before pfClusters are formed.  I will merge your code, because I think it is in good shape, but I think we should discuss this later today.